modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/FormBuilder.es.js

```diff
@@ -101,6 +110,16 @@ export const FormBuilder = () => {
 
 	const validateFormWithObjects = useValidateFormWithObjects();
 
+	const removeErrorMessage = useCallback(
+		(index) => {
+			const errorMessages = [...errorList].splice(index, 1);
+
+			setErrorList(errorMessages);
+		},
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
@miltonmc ,
I needed to add this, otherwise I would need to add `errorList` to the dependency array and it breaks the container's close button logic.
What do you think?
What do you think?

Reply:
Ideally we should not do that, but sometimes it is necessary.

I didn't understand why add `errorList` is breaking the `onClose` in this case :confused: 
We can try explore other options together if you want